SECURITY.md

```diff
@@ -0,0 +1,59 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, either:
+
+1. Report it on Github directly you can follow the procedure described
+[here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)
+and:
+    - Navigate to the security tab on the repository
+    - Click on 'Advisories'
+    - Click on 'Report a vulnerability'
+
+2. Send an email to `security@crossplane.io` detailing the issue and steps
+to reproduce.
+
+The reporter(s) can expect a response within 24 hours acknowledging
+the issue was received. If a response is not received within 24 hours, please
+reach out to any committer directly to confirm receipt of the issue.
```

Comment:
24 business hours? If we really do mean 24 hours we'll need to attach this email to a pager rotation.

Reply:
I'm OK with this, with the language update to "can typically expect a response within 24 hours acknowledging..."

Since this is a typical expectation (not a promise), and there is a fallback (ping a maintainer directly), I am OK with not having a pager rotation for this.  I've tried to be very responsive for security disclosures on other projects in the past, and giving an initial ACK quickly is something I'm OK with, since it's not a SLA or guarantee.

Reply:
changed wording as suggested